.claude/commands/init.md

```
@@ -0,0 +1,170 @@
+---
+name: init
+description: Initialize the Codebase Singularity framework for a new project
+arguments:
+  - name: template
+    description: "Optional template: basic, nextjs, express, django, fastapi"
+    required: false
+---
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)